stages/org.osbuild.greenboot.conf

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+"""
+update greenboot configuration in /etc/greenboot/greenbot.conf
+"""
+
+import sys
+from configparser import RawConfigParser
+import osbuild.api
+
+SCHEMA_2 = r"""
+"required": ["config"],
+"additionalProperties": false,
+"properties": {
+    "config": {
+      "type": "object",
+      "properties": {
+        "max_boot_attempt": {
+          "type": "int",
+          "description": "The maximum number of boot attempts before rollback",
+          "default": 3
+        },
+        "watchdog_check": {
+          "type": "boolean",
+          "description": "Validation for the presence of real time clock",
+          "default": true
+        },
+        "watchdog_grace_period": {
+          "type": "int",
+          "description": "This variable is the number of hours after an upgrade that we consider the new deployment as culprit of reboot",
+          "default": 24
+        },
+        "service_monitor": {
+          "type": "string",
+          "description": "Validates the status of the services before boot-complete.target is reached",
+          "default": "sshd NetworkManager"
```

Comment:
Do we have an agreement on where to take default-values from? This configuration now suggests that _iff_ you use this stage, but without specific configurations, the defaults configured _in this stage_ will be applied. This feels weird to me. The stages should not encode policy, right?

Preferably, we would resort to the default values of greenboot itself, or make the caller of this stage ensure to provide suitable defaults. Just imagine trying to figure out where the value in your final image comes from. It might now be a value from greenboot itself, or from the caller of this stage, or just the default of this stage. Ignoring user interaction and/or longer chains on the call-site.

Reply:
these should definitely come from "above" as in from osbuild-composer, please drop them @say-paul